AirCasting/SDSync/ViewsFlow/SDSyncUnplugAB/UnplugABViewModel.swift

```diff
@@ -0,0 +1,29 @@
+// Created by Lunar on 08/12/2021.
+//
+
+import Foundation
+
+protocol UnplugABViewModel: ObservableObject {
+    var presentNextScreen: Bool { get set }
```

Comment:
after making changes Janek asked for this should become get-only (an issue with navigation link will arise, but you can refer to other PRs on how to handle it)